tests/src/python/test_qgsfileutils.py

```diff
@@ -92,6 +92,60 @@ def testFindClosestExistingPath(self):
         self.assertEqual(QgsFileUtils.findClosestExistingPath(sub_folder2 + '/../subfolder2'), sub_folder2)
         self.assertEqual(QgsFileUtils.findClosestExistingPath(sub_folder2 + '/../subfolder2/zxcv/asfdasd'), sub_folder2)
 
+    def testAutoFinder(self):
+        temp_folder = tempfile.mkdtemp()
+        base_folder = os.path.join(temp_folder, 'base_level')
+        os.mkdir(sub_fold1)
+        side_fold = os.path.join(temp_folder, 'sidefold')
+        os.mkdir(side_fold)
+        nest = os.path.join(sub_fold1, 'direct_nest')
+        os.mkdir(nest)
+        side_nest = os.path.join(side_fold, 'side_nest')
+        os.mkdir(side_nest)
+        filename = "findme.txt"
+
+        #unexisting
+        files = QgsFileUtils.findFile(filename, nest, 1, 4)
+        print(files, '105')
```

Comment:
I think you either want to remove the prints completely or replace them with something that gives more context than outdated line numbers ;)

Reply:
I just want to see if the test passes with the renaming that I did, this will help me find any mistakes on my end, once all tests are good I'll remove them.

Reply:
ok, no worries

Reply:
Done, all tests are ok, forgot to sip, should be fine now.